tests/e2e/main_test.go

```diff
@@ -66,10 +66,15 @@ func TestMain(m *testing.M) {
 		"",
 		"DCGM-exporter image tag to use")
 
+	flag.StringVar(&suiteCfg.arguments,
+		"arguments",
+		"",
+		`DCGM-exporter command line parameters. Example: -arguments={-f=/etc/dcgm-exporter/default-counters.csv}`)
+
 	flag.Parse()
 	os.Exit(m.Run())
 }
-
+git
```

Comment:
may be an accident.

Reply:
Oops, @rohit-arora-dev , thank you for finding it. I have already pushed the change.